installers/olm/description.openshift.md

```diff
@@ -10,136 +10,47 @@ providing the essential features you need to keep your PostgreSQL clusters up an
+  Set your [pod anti-affinity][k8s-anti-affinity], node affinity, Pod tolerations and more rules to customize your deployment topology!
+- **Full Customizability**: Crunchy PostgreSQL for Kubernetes makes it easy to get your own PostgreSQL-as-a-Service up and running
+  and fully customize your deployments, including:
+    - Choose the resources for your Postgres cluster: [container resources and storage size][resize-cluster]. [Resize at any time][resize-cluster] with minimal disruption.
     - Use your own container image repository, including support `imagePullSecrets` and private repositories
-    - Bring your own trusted certificate authority (CA) for use with the Operator API server
-    - Override your PostgreSQL configuration for each cluster
+    - [Customize your PostgreSQL configuration][customize-cluster]
 
 and much more!
 
-[disaster-recovery]: https://access.crunchydata.com/documentation/postgres-operator/latest/architecture/disaster-recovery/
-[high-availability]: https://access.crunchydata.com/documentation/postgres-operator/latest/architecture/high-availability/
-[pgo-create-cluster]: https://access.crunchydata.com/documentation/postgres-operator/latest/pgo-client/reference/pgo_create_cluster/
-[provisioning]: https://access.crunchydata.com/documentation/postgres-operator/latest/architecture/provisioning/
+[backups]: https://access.crunchydata.com/documentation/postgres-operator/v5/tutorial/backups/
+[clone]: https://access.crunchydata.com/documentation/postgres-operator/v5/tutorial/disaster-recovery/#clone-a-postgres-cluster
+[customize-cluster]: https://access.crunchydata.com/documentation/postgres-operator/v5/tutorial/customize-cluster/
+[disaster-recovery]: https://access.crunchydata.com/documentation/postgres-operator/v5/tutorial/disaster-recovery/
+[high-availability]: https://access.crunchydata.com/documentation/postgres-operator/v5/tutorial/high-availability/
+[monitoring]: https://access.crunchydata.com/documentation/postgres-operator/v5/tutorial/monitoring/
+[pool]: https://access.crunchydata.com/documentation/postgres-operator/v5/tutorial/connection-pooling/
+[provisioning]: https://access.crunchydata.com/documentation/postgres-operator/v5/tutorial/create-cluster/
+[resize-cluster]: https://access.crunchydata.com/documentation/postgres-operator/v5/tutorial/resize-cluster/
+[tls]: https://access.crunchydata.com/documentation/postgres-operator/v5/tutorial/customize-cluster/#customize-tls
 
 [k8s-anti-affinity]: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#inter-pod-affinity-and-anti-affinity
+[k8s-nodes]: https://kubernetes.io/docs/concepts/architecture/nodes/
 
 [pgBackRest]: https://www.pgbackrest.org
+[pgBouncer]: https://access.crunchydata.com/documentation/postgres-operator/latest/tutorial/pgbouncer/
 [pgMonitor]: https://github.com/CrunchyData/pgmonitor
 
 
-## Before You Begin
+## Post-Installation
```

Comment:
The rest of this document is effectively a single link now. ❓ Should it be some subset of the tutorial instead?

Reply:
I would just link back to our tutorial.